wiki/How_You_Can_Help!/en.md

```diff
@@ -1,80 +1,78 @@
 
-Here is a current list of areas I can think of where anyone can help out, ordered by the most important in my eyes. I'll try and keep this updated as I see things come up, but don't hesitate to make suggestions (preferably in this thread) if you have any.
+Read more about how you can get started contributing to the wiki [here!](https://osu.ppy.sh/help/wiki/osu!_wiki_Contribution_Guide)
 
-## Help out with [the wiki](https://github.com/ppy/osu-wiki "osu-wiki in GitHub")!
+## Help keep osu! friendly and safe
 
-In order to keep the content on the website as fresh as possible, I'd like to move all sections containing text to the wiki. This includes FAQ, ranking guidelines, all our forum-based guides, the history/information sections. While we have made huge progress already (thanks to a few very dedicated users), keeping a wiki up-to-date is an ongoing process which basically anyone can help with. There is also a lot of translation opportunities remaining to make the content accessible in every language known to mankind!
+It's not always sunshine in the osu! community, so we have a dedicated team of moderators to keep the place comfy and safe. However, the moderation team keeps track of all 25 channels (and more!) in the chat, and a countless entries on the forums.
 
-### Where to start:
+Fortunately, it's still easy for you to help out with the community, here's some things you can do to help:
 
--   Go through all the FAQ and other information sections on the osu! site, and make sure the info is easily accessible (and complete) on the wiki. If it isn't, add it in (you can begin by just copying, pasting and reformatting). Make an Issue to discuss any changes if you are not confident about it, or go straight ahead to do a Pull Request and update old information you come across.
--   Watch the [Pull Requests](https://github.com/ppy/osu-wiki/pulls "Pull Requests") as often as you can (bookmark it :p), and make sure people aren't adding useless or incorrect information by reviewing. Keep a tab on what is being worked on and chip in if you can!
--   Make sure the information on the wiki is as concise, clear and beautiful as possible. Tidy up any formatting issues you find, and push people in the right direction where required.
--   Make this post into a wiki page :D! Translate it to more languages — we need help from more than just English writers (fact: over 75% of the osu! community doesn't speak English and/or understand English fully).
+- Follow your favorite channels at the osu! in-game chat, or at the forums in the web.
+- If you see something happening which you think feels inappropriate or disturbs you in any way, use the `!report` command at the in-game chat to notify moderators of what's happening.
+    - Don't worry about not having your report not being read, the moderators will always read your reports.
 
-## Help keep osu! friendly and safe!
+## Help answer technical, beatmap, and gameplay support
```

Comment:
Header sounds weird, forgot a word here?
maybe generalise this to something like "Help by supporting other community members"